### DISCUSSION TITLE
Getting connected to the group repos

### DISCUSSION THREAD

**Discussion by TaylorHardaway** [nss-evening-cohort-14/evening-client-side#53] [Help Me!] [answered]
Taylor Hardaway
-I don't see any other discussions on this.
-After working with some other people it seems that others are able to connect to the group repos in a way that I am not.  One person had a link under their profile picture that I do not, and another said that that I should have an organizations option from the profile dropdown in the upper righthand corner. I believe that I didn't accept an invitation that I should have.
-There are no error messages, just links that I don't have.
-  https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-an-individuals-access-to-an-organization-repository
    https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories
     https://git-scm.com/book/en/v2/GitHub-Managing-an-organization
- https://github.com/TaylorHardaway

**Comment by drteresavasquez** [comment, marked as answer]
Not sure I understand. You should move your card over in CodeTracker and it creates a repo for you. Follow the steps to clone **YOUR** repo down so that you can get to work. At this point, you should know how to clone a repo to your local machine. If not, I would say you need to search for that process or follow one of the class videos.

**Reply by TaylorHardaway** [reply]
Yeah, I'm able to do that (although some of us seem to push to the group repo by accident, but we got through it), when a couple of students have been explaining some stuff to me they are getting to a step where they want to go to the group repo's (last night it was to copy some code from our last class) and I just don't have the links that they have. It may not actually be that big of a deal, but I just thought I would ask in case it is somewhere down the road it is.
